core/src/Platforms/iOS/WebviewBase.cs

```diff
@@ -28,14 +28,19 @@
 using System.Threading.Tasks;
 using SafariServices;
 using Foundation;
+using AuthenticationServices;
+using UIKit;
 
 namespace Microsoft.Identity.Core.UI
 {
     internal abstract class WebviewBase : NSObject, IWebUI, ISFSafariViewControllerDelegate
     {
         protected static SemaphoreSlim returnedUriReady;
         protected static AuthorizationResult authorizationResult;
+        protected static UIViewController viewController;
```

Comment:
should viewController be static?  the other view controllers and sessions are instance-level and not static.

Reply:
I only asked because the other view controllers weren't static.  if this blows things up based on current design, ignore for now and you've done the right thing here.

Reply:
ok...sounds good. I'll leave for now. Thanks.